package.json

```diff
@@ -14,7 +14,8 @@
     "build:showcase": "npm run opts -- webpack --config-name showcase",
     "showcase": "II_FETCH_ROOT_KEY=1 npm run opts -- webpack serve --config-name showcase",
     "screenshots": "npm run opts -- ./src/frontend/screenshots.ts",
-    "test": "jest --maxWorkers 1 --roots ./src/frontend --verbose --testPathIgnorePatterns=\"src/frontend/src/test-e2e\"",
+    "test": "jest",
```

Comment:
by default, won't this run all units + integration tests, i.e. more tests than `npm run test:all`?

Also it seems to be running a lot of third party tests:

```
~/internet-identity$ npm run jest

> @dfinity/internet-identity@0.1.0 jest
> jest

jest-haste-map: Haste module naming collision: @dfinity/service-worker
  The following files share their name; please adjust your hasteImpl:
    * <rootDir>/.home/.cargo/git/checkouts/ic-ca00ee28655ee32e/5248f11/typescript/service-worker/pack
age.json
    * <rootDir>/.home/.cargo/git/checkouts/ic-ca00ee28655ee32e/7919a75/typescript/service-worker/pack
age.json


 RUNS  ...checkouts/ic-ca00ee28655ee32e/a7b1978/typescript/service-worker/src/sw/http_request.test.ts
 RUNS  ...eckouts/ic-ca00ee28655ee32e/01f31f4/typescript/service-worker/src/sw/requests/index.test.ts
 RUNS  ...checkouts/ic-ca00ee28655ee32e/d4729f7/typescript/service-worker/src/sw/http_request.test.ts
 RUNS  ...checkouts/ic-ca00ee28655ee32e/a7ab323/typescript/service-worker/src/sw/http_request.test.ts
 RUNS  ...checkouts/ic-ca00ee28655ee32e/a647ea4/typescript/service-worker/src/sw/http_request.test.ts
 RUNS  ...checkouts/ic-ca00ee28655ee32e/3635513/typescript/service-worker/src/sw/http_request.test.ts
 RUNS  ...checkouts/ic-ca00ee28655ee32e/7b21f56/typescript/service-worker/src/sw/http_request.test.ts
```

Reply:
Two options:

- either we spent time trying to fix the configuration so that `npm run test` is set per default and don't do this
- or we keep `test` to run all tests and introduce a new shorhand `test:only` that has the caveat you mention

wdyt?

Reply:
How about adding a `jest` script that calls jest? or how about running `npx jest` simply? That being said, I think it wouldn't be a bad idea to fix the config, but probably out of scope for this PR